### PR TITLE
Deposit aleksandr: Described one method for Deposit page

### PR DESCRIPTION
**Pull request by LeonadO555**

### CODE REVIEW

File: src/test/java/pages/customerLogin/account/deposit/DepositPage.java [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+import wait.Wait;
 
 public class DepositPage extends PageBase {
     public DepositPage(WebDriver driver) {
         super(driver);
     }
+
+    Wait wait;
+    @FindBy(xpath = "//input[@placeholder='amount']")
+    protected WebElement amountField;
```
Comment (LeonadO555) [inline src/test/java/pages/customerLogin/account/deposit/DepositPage.java:20]: Rename please on fieldAmount

File: src/test/java/pages/customerLogin/account/deposit/DepositPage.java [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
 import pages.PageBase;
+import wait.Wait;
 
 public class DepositPage extends PageBase {
     public DepositPage(WebDriver driver) {
         super(driver);
     }
+
+    Wait wait;
+    @FindBy(xpath = "//input[@placeholder='amount']")
```
Comment (LeonadO555) [inline src/test/java/pages/customerLogin/account/deposit/DepositPage.java:19]: jsd;klfjslk;JDF

File: src/test/java/pages/customerLogin/account/deposit/DepositPage.java (hunk 1 above)
Comment (LeonadO555) [reply src/test/java/pages/customerLogin/account/deposit/DepositPage.java:20]: okay i do it

File: src/test/java/pages/customerLogin/account/deposit/DepositPage.java (hunk 2 above)
Comment (dushuk) [reply src/test/java/pages/customerLogin/account/deposit/DepositPage.java:19]: ok

File: src/test/java/pages/customerLogin/account/deposit/DepositPage.java (hunk 1 above)
Comment (LeonadO555) [reply src/test/java/pages/customerLogin/account/deposit/DepositPage.java:20]: I did it

**Comment by LeonadO555** [comment]
This PR is not valid for me , I will close it

**Comment by LeonadO555** [comment]
I will do new realization for &&&& method

**Pull request merged**